kbase-extension/static/kbase/js/widgets/narrative_core/upload/importErrors.js

```diff
@@ -137,6 +144,26 @@ define(['common/ui', 'common/html'], (UI, html) => {
                             message: error.message,
                         });
                         break;
+                    case BULK_SPEC_ERRORS.NOT_BULK_TYPE:
```

Comment:
These really only come from the custom code in ImportSetup. I'm not sure it's worth dealing with edge cases like missing files at this point, as nothing can trigger that.

Reply:
Unless, of course, the "valid" response from the staging area actually loses the file names? I don't think that can happen, though.